eigen.cmake

```diff
@@ -17,24 +17,34 @@
 # -----------------
 #
 # This macro gets eigen include path from pkg-config file, and adds it include directories.
+# On Windows, there is no pkg-config for Eigen, thus we fall back on a manual search.
```

Comment:
This is out of date given the recent change
 Please update it to reflect the current strategy